atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -115,6 +115,17 @@ def set_koji_image_build_data(self):
                 )
             raise ValueError(err_msg)
 
+        build_type = self.session.getBuildType(self.koji_build['build_id'], strict=True)
+        if 'image' not in build_type:
```

Comment:
This also means operator images will not get source builds. This may be OK, just making sure we are aware of this here.

Reply:
Not really, a build type would come with both types for operator builds, so this should be safe.

Reply:
yeah 'image' type will be there